assets/lang/ko_KR.ini

```diff
@@ -1225,7 +1225,7 @@ Clear UI background = UI 배경 지움
 Confirmation Button = 확인 버튼
 Date Format = 날짜 형식
 Day Light Saving = 서머 타임
-DDMMYYYY = DDMMYYYY
+DDMMYYYY = YYYYMMDD
```

Comment:
This won't change the date format, it'll just confuse the user because the other options will show YYYYMMDD when they're not.

-[Unknown]